README.md

```diff
@@ -442,7 +442,7 @@ To ensure that a specific set of permissions are absent from the ACL, set `purge
 
  * When using SIDs for identities, autorequire tries to match to users with fully qualified names (e.g., User[BUILTIN\Administrators]) in addition to SIDs (User[S-1-5-32-544]). However, it can't match against 'User[Administrators]', because that could cause issues if domain accounts and local accounts share the same name e.g., 'Domain\Bob' and 'LOCAL\Bob'.
 
- * When referring to accounts in the `APPLICATION PACKAGE AUTHORITY`, use either their SID values or their unqualified names. The Windows API has well documented bugs preventing the fully qualifed account names from being used.
+ * When referring to accounts in the `APPLICATION PACKAGE AUTHORITY`, with Puppet version older than `6.22.0` or `7.5.0`, use either their SID values or their unqualified names. The Windows API has well documented bugs preventing the fully qualifed account names from being used.
```

Comment:
with Puppet version**s** older than... sounds better to me. Also I don't think we need to wrap the versions in backticks but now I'm just nitpicking.